src/ops/ops.go

```diff
@@ -195,6 +196,21 @@ func (o *ops) Reboot() error {
 	return nil
 }
 
+func (o *ops) SetEfiBootMgr(device string) error {
+	o.log.Info("Setting EFI Boot Manager to boot from disk")
```

Comment:
I think you should first check if the host is booting from efi.
```suggestion
        _, err := o.ExecPrivilegeCommand(o.logWriter, "test", "-f", "/sys/firmware/efi")
	if err != nil {
		o.log.Info("efi not supported")
		return nil
	}
	o.log.Info("Setting EFI Boot Manager to boot from disk")
```